src/ovs_stats.c

```diff
@@ -0,0 +1,982 @@
+    oconfig_item_t *child = ci->children + i;
+    if (strcasecmp("Address", child->key) == 0) {
+      if (cf_util_get_string_buffer(child, ovs_stats_cfg.ovs_db_node,
+                                    OVS_DB_ADDR_NODE_SIZE) != 0) {
+        ERROR("%s: parse '%s' option failed", plugin_name, child->key);
+        return (-1);
+      }
+    } else if (strcasecmp("Port", child->key) == 0) {
+      if (cf_util_get_string_buffer(child, ovs_stats_cfg.ovs_db_serv,
+                                    OVS_DB_ADDR_SERVICE_SIZE) != 0) {
+        ERROR("%s: parse '%s' option failed", plugin_name, child->key);
+        return (-1);
+      }
+    } else if (strcasecmp("Socket", child->key) == 0) {
+      if (cf_util_get_string_buffer(child, ovs_stats_cfg.ovs_db_unix,
+                                    OVS_DB_ADDR_UNIX_SIZE) != 0) {
+        ERROR("%s: parse '%s' option failed", plugin_name, child->key);
+        return (-1);
+      }
+    } else if (strcasecmp("Bridges", child->key) == 0) {
+      for (int j = 0; j < child->values_num; j++) {
+        /* check value type */
+        if (child->values[j].type != OCONFIG_TYPE_STRING) {
+          ERROR("%s: Wrong bridge name [idx=%d]. "
+                "Bridge name should be string",
+                plugin_name, j);
+          goto cleanup_fail;
+        }
+        /* get value */
+        if ((br_name = strdup(child->values[j].value.string)) == NULL) {
+          ERROR("%s: strdup() copy bridge name fail", plugin_name);
+          goto cleanup_fail;
+        }
+        if ((bridge = ovs_stats_get_bridge(g_monitored_bridge_list_head,
+                                           br_name)) == NULL) {
+          if ((bridge = calloc(1, sizeof(bridge_list_t))) == NULL) {
+            ERROR("%s: Error allocating memory for bridge", plugin_name);
+            goto cleanup_fail;
+          } else {
+            pthread_mutex_lock(&g_stats_lock);
+            /* store bridge name */
+            bridge->name = br_name;
+            bridge->next = g_monitored_bridge_list_head;
+            g_monitored_bridge_list_head = bridge;
+            pthread_mutex_unlock(&g_stats_lock);
+            DEBUG("%s: found monitored interface \"%s\"", plugin_name, br_name);
+          }
+        }
+      }
+    } else {
+      WARNING("%s: option '%s' not allowed here", plugin_name, child->key);
+      goto cleanup_fail;
+    }
+  }
+  return (0);
+
+cleanup_fail:
+  ovs_stats_free_bridge_list(g_monitored_bridge_list_head);
+  return (-1);
+}
+
+/* Initialize OvS Stats plugin*/
+static int ovs_stats_plugin_init(void) {
+  ovs_db_callback_t cb = {.post_conn_init = ovs_stats_initialize,
+                          .post_conn_terminate = ovs_stats_conn_terminate};
+
+  INFO("%s: Connecting to OVS DB using address=%s, service=%s, unix=%s",
+       plugin_name, ovs_stats_cfg.ovs_db_node, ovs_stats_cfg.ovs_db_serv,
+       ovs_stats_cfg.ovs_db_unix);
+  /* connect to OvS DB */
+  if ((ovs_db = ovs_db_init (ovs_stats_cfg.ovs_db_node,
```

Comment:
ovs_db is static. You're using the g_ prefix for all other statics, I believe you should do the same here.